csrc/includes/cublas_wrappers.h

```diff
@@ -29,7 +30,8 @@ int cublas_gemm_ex(cublasHandle_t handle,
                    const float* A,
                    const float* B,
                    float* C,
-#ifdef __HIP_PLATFORM_AMD__
+// TODO HIP: Remove backward compatibility for torch<=2.0 in future
+#if defined(__HIP_PLATFORM_AMD__) && TORCH_VERSION_MAJOR <= 2 && TORCH_VERSION_MINOR <= 0
```

Comment:
Won't this condition not work properly, since version 1.13 for example would yield false the minor version check wouldn't do the right thing?

Reply:
Sorry, let me correct it now.